src/content/learn/start-a-new-react-project.md

```diff
@@ -26,7 +26,12 @@ npx create-next-app
 
 If you're new to Next.js, check out the [Next.js tutorial.](https://nextjs.org/learn/foundations/about-nextjs)
 
-Next.js is maintained by [Vercel](https://vercel.com/). You can [deploy a Next.js app](https://nextjs.org/docs/deployment) to any Node.js or serverless hosting, or to your own server. [Fully static Next.js apps](https://nextjs.org/docs/advanced-features/static-html-export) can be deployed to any static hosting.
+Next.js is maintained by [Vercel](https://vercel.com/). You can [deploy a Next.js app](https://nextjs.org/docs/pages/building-your-application/deploying) to any Node.js or serverless hosting, or to your own server. [Fully static Next.js apps](https://nextjs.org/docs/pages/building-your-application/deploying/static-exports) can be deployed to any static hosting.
+
+<Note>
+There are two ways of building a Next.js app - using [*App Router*](#nextjs-app-router) or *Pages Router*. During installation, you will be prompted to use the *App Router*. Choose *'No'* to stick with the older *Pages Router*. However, using an *App Router* is the current recommended way of building a Next.js app. The *App Router* is built on bleeding-edge React features like **React Server Components**
+See [Bleeding Edge React Frameworks](#bleeding-edge-react-frameworks).
+</Note>
```

Comment:
I feel like this wording could be confusing, as it goes back and forth between suggesting app router and pages router, and it starts by explaining "no" even though "yes" is recommended.